platform/src/abstractions/Platform.Search/SearchConnection.cs

```diff
@@ -3,14 +3,22 @@
 using Azure.Search.Documents;
 using Azure.Search.Documents.Models;
 
-namespace Platform.Infrastructure.Search;
+namespace Platform.Search;
+
+public interface ISearchConnection<T>
+{
+    Task<(long? Total, IEnumerable<ScoreResponse<T>> Response)> SearchAsync(string? search, string? filters, int? size = null);
+    Task<T> LookUpAsync(string? key);
+    Task<SearchResponse<T>> SearchAsync(SearchRequest request, Func<FilterCriteria[], string?>? filterExpBuilder = null, string[]? facets = null);
+    Task<SuggestResponse<T>> SuggestAsync(SuggestRequest request, Func<string?>? filterExpBuilder = null,string[]? selectFields = null);
```

Comment:
chore: Formatting issue should get picked up with a `dotnet format`

Reply:
Sorted